chapters/connectors.tex

```diff
@@ -904,7 +904,17 @@ \subsection{Connection Graphs and Their Operators}\label{overconstrained-equatio
 \subsection{Generation of Connection Graph Equations}\label{generation-of-connection-graph-equations}
 
 Before connection equations are generated, the virtual connection graph is transformed into a set of spanning trees by removing optional spanning tree edges from the graph.
-This is performed in the following way:
+
+As a preparation of generating connection graph equations, any redundant breakable branches are removed in each connection set:
+\begin{enumerate}
+\item
+  Build a spanning tree for the connection set.
```

Comment:
I guess this is the part that https://github.com/modelica/ModelicaSpecification/issues/3370#issuecomment-1545525843 suggests to change?

Reply:
Yes, those extra spanning trees and removed redundant connections correspond to the suggestion in that comment.

Reply:
In that case, aren't we missing that this should be a spanning tree of breakable branches only?

Reply:
I think there's too much risk of ambiguity to leave the detail out, in the current context where we also have other types of edges connecting the connection set vertices.
```suggestion
  Build a spanning tree for the connection set, using only breakable branches.
  (By the connection set definition, a spanning tree always exists.)
```